ASWF/meetings/tsc/2020-08-17.md

```diff
@@ -0,0 +1,121 @@
+    - Patrick: Could it be an option to make it a dynamic lib released as 
+      utils?
+    - Mark B: Small amount of users will use these. Don't need the extra 
+      library to do anything. Most people will encounter OCIO through a package 
+      manager or as an end user, and just want to run OCIO. Won't need all the 
+      extra bits. From our lib side, it's about shrinking the amount of stuff 
+      that we're supporting. Good to keep it small. Most libs don't come with 
+      utils.
+    - Patrick: Valid point, but fear people will rewrite these pieces. They 
+      will help ease transition to v2.
+    - Mark B: If that stuff is good, should have a space in libOpenColorIO.so, 
+      part of namespace with utils.
+    - Michael: Like OIIO::ImageBufAlgo.
+    - Patrick: Want to limit dependencies. libutils depends on OIIO and other 
+      libs, which is why it was decided to make it external. Want to avoid 
+      people rewriting these pieces.
+    - Mark B: Think it's good to provide, but shared lib probably not right way 
+      to do it. Provide code and allow implementors to decide how they want to 
+      implement and what dependencies to use. We've made it easy to get started.
+    - Patrick: If we don't distribute, package managers will not have these 
+      pieces.
+    - Mark B: Most people downloading lib wont need this.
+    - Doug: There are different categories. Some things could be separate, but 
+      apputils probably needs to move into the core library.
+    - **TODO**: Patrick will investigate alternative options and report back 
+      when there's time.
+
+* PR #1104:
+  - Patrick: Fix for cmake detection via CMake user registry, could use review 
+    by someone else. Special case, but should be merged soon.
+
+* Python optional parameters:
+  - Patrick: Thomas discovered some Python interface that should have default 
+    parameters, which were previously optional parameters are now mandatory 
+    (issue #1105). C++ has no optional parameters per OCIO design, but this 
+    rule is not necessarily appropriate for Python. How should we handle this 
+    in Python? Also highlights being backwards compatible for C++ vs Python. 
+    We should decide if we are backwards compatible and consistent with v1 
+    Python API.
+  - Mark: Perhaps both can change if the APi is bad.
+  - Patrick: Think we need to be consistent where possible.
+  - Mark T: Hit the same issue while testing at Foundry. Differences in 
+    default parameters of API were problematic.
+  - Michael: There have also been some issues where null pointers in the C++ 
+    API don't interface well with None in Python and cause a crash. pybind11
+    provides a solution for this which we've been fixing whenever a case is 
+    found.
+  - Doug: We also made improvements in C++ to not crash in these cases.
+  - TSC agrees that optionals args in the Python API (contrary to the c++ API)
```

Comment:
Excellent, the opposite would have really been sulked by people using the Python bindings (and probably caused a revolution :)